packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -312,13 +312,17 @@ async function deleteScratchOrg(
 
   try {
     let communityUrl = '';
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (isCi()) {
+      runner.add(async (log) => await authorizeDevOrg(log, options));
+    }
     if (options.deleteOldOrgs) {
       runner.add(async (log) => {
         await deleteOldOrgs(log, options);
       });
     }
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (isCi()) {
+      runner.add(async (log) => await authorizeDevOrg(log, options));
+    }
```

Comment:
I mean is authorizing the dev org  the for the second time really necessary?
```suggestion
```

Reply:
I reckon yes because deleteOldOrgs do connect to other org I reckon.

Reply:
Could try without tho, yolo.
If it works gg